lib/smart_answer_flows/am-i-getting-minimum-wage.rb

```diff
@@ -49,7 +51,7 @@ def define
         end
       end
 
-      use_shared_logic "minimum_wage"
+      append(Shared::MinimumWageFlow.build)
```

Comment:
> Do you know why you wouldn't have used them in this case?

@floehopper: In this specific case I think I would've left them out to be consistent with calls to the other DSL methods within the `define` method (e.g. `content_id`, `name`, `multiple_choice`). In the more general case, I'm not sure I'd be able to explain how I decide whether to use them or not.

I didn't know about that section of the Ruby Style Guide so I'll bear that in mind for the future.